services/src/main/scala/cromwell/pipeline/service/ProjectFileService.scala

```diff
@@ -0,0 +1,17 @@
+package cromwell.pipeline.service
+
+import cats.instances.string._
+import cats.syntax.foldable._
+import cromwell.pipeline.datastorage.dto.{ FileContent, ValidateError }
+import cromwell.pipeline.womtool.WomToolAPI
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+class ProjectFileService(womTool: WomToolAPI)(implicit executionContext: ExecutionContext) {
+
+  def validateFile(fileContent: FileContent): Future[Either[ValidateError, Unit]] =
+    Future(womTool.validate(fileContent.content)).map {
+      case Left(value) => Left(ValidateError(value.mkString_("\n")))
```

Comment:
Why are we making one string instead of passing array of validation errors?

Reply:
Because I want to show all errors in html body response

Reply:
But if you're passing list, list will be marshalled to JSON array. 
Rest api client will get not one string, but list of errors.

Reply:
Seems legit) Will fix it, thanks

Reply:
I remembered that I wanted to return Error when file is invalid, so I extended Error, but constructor of Error contains only String type param.

Reply:
You're not forced to use `Exception`'s constructor. If you don't need it -- don't use it.

Reply:
Fixed it, thanks